contrib/Dockerfile.manylinux

```
@@ -342,9 +348,12 @@ RUN rm -rf $VIRTUAL_ENV && uv venv $VIRTUAL_ENV --python $DEFAULT_PYTHON_VERSION
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 # Install python dependencies
 RUN uv pip install --upgrade meson meson-python pybind11 patchelf pyYAML click setuptools tabulate auditwheel tomlkit
-# Install PyTorch
+# Install the default PyTorch used by the outer NIXL build. Per-version NIXL EP
+# extensions are built later in isolated environments by build-wheel.sh.
+# torch 2.11 is not available on the cu132 index; 2.13 is the current default.
+ARG DEFAULT_TORCH_VERSION="2.13"
 RUN export UV_INDEX="https://download.pytorch.org/whl/cu$(echo $CUDA_VERSION | cut -d. -f1,2 | tr -d .)" && \
```

Hadolint (2.15.1)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)